packages/MainUI/src/components/navigation.tsx

```diff
@@ -82,6 +82,8 @@ const Navigation: React.FC = () => {
         roles={[]}
```

Comment:
Please avoid declarations in JSX.
In this case, you could declare the constant outside of this component and reference it here.
```tsx

const roles = [];

export const MyCmp = (props) => <TheChild roles={roles}>...</>
```